src/txpasslib/_threads.py

```diff
@@ -0,0 +1,24 @@
+from twisted.internet.defer import Deferred
+from twisted.python.failure import Failure
+
+
+def defer_to_worker(deliver, worker, work, *args, **kwargs):
+    """
+    Run a task in a worker, delivering the result as a ``Deferred`` in the
```

Comment:
epytext unfortunately means we can't use RTD, but as discussed on IRC, we can probably cobble something together using Travis, tox, pydoctor, and S3 to replace RTD.